### ISSUE TITLE
Pagespeed Optimization blank.gif

### ISSUE THREAD

**Issue by jowe1** [benjaminkott/bootstrap_package#489]
This is not really a bug but would a big improvement with little effort.
The currently used blank.gif that is a placeholder for the real images that are being loaded later on, is not optimized in terms of size. With just 1 KB this might not matter much to actual pagespeed but google pagespeed insights is still analyzing it and the check says: 

> By compression of ...Resources/Public/Images/blank.gif you could save 1 KB (93 %).

I already downloaded the compressed file from google:
![blank](https://user-images.githubusercontent.com/19427513/33543496-8cad9bf4-d8d7-11e7-8969-4e1e0746c244.png)
All you need to change then is the Image.html:
`<img src="{f:uri.image(src: 'EXT:bootstrap_package/Resources/Public/Images/**blank.png**')}" data-src="{f:uri.image(image: file)}" data-large="{f:uri.image(image: file, cropVariant: 'default', maxWidth: 1140)}" data-medium="{f:uri.image(image: file, cropVariant: 'medium', maxWidth: 940)}" data-small="{f:uri.image(image: file, cropVariant: 'small', maxWidth: 720)}" data-extrasmall="{f:uri.image(image: file, cropVariant: 'extrasmall', maxWidth: 320)}" title="{file.title}" alt="{file.alternative}" class="lazyload" data-preload="{f:if(condition: settings.preload.images, then: 'true', else: 'false')}" />`
And of course delete blank.gif and put blank.png into the repository as its replacement.

Would be cool if you could do this, unfortunately I was not able to push to the repository thus I could not create a pull request for the changes. Thank you.

**Comment by benjaminkott** [comment]
@jowe1 hey, what version are you using, just checked the current version and pagespeed does not mention this file, also the image optimizer script shows that everything should be fine.

![image](https://user-images.githubusercontent.com/3243119/33561571-4ca09aa2-d913-11e7-9a2b-067494fabc93.png)

**Comment by jowe1** [comment]
@benjaminkott I am currently using 8.0.3 and have just seen that there is 9.0.0 now. I will be updating to it asap. Thanks for letting me know!

**Comment by jowe1** [comment]
@benjaminkott I recently updated to v9 and everything looks fine, now.
As a matter of fact we optimized the site to have 100/100 on Google Pagespeed Insights and according to pingdom is in the top 3% of tested websites regarding its performance.

Is there a way to promote this as an outstanding project? It is running in a docker environment hosted in Amazon AWS cloud and it is really fast. Thank you so much!
